File: backend/ProductApi/Order.cs
public class Order
{
    public int Id { get; set; }
    public List<int> ProductIds { get; set; } = new List<int>();
    public decimal TotalPrice { get; set; }
    public string Status { get; set; } = "Pending"; // Status: Pending, Shipped, Delivered
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: src/types/Order.ts
export interface Order {
    id: number;
    products: {
        id: number;
        name: string;
        price: number;
        quantity: number;
        imageUrl?: string;
    }[];
    totalPrice: number;
    status: "Pending" | "Shipped" | "Delivered";
    createdAt: string;
}
